src/tree_schema.h

```diff
@@ -523,7 +523,9 @@ struct lysp_stmt {
 struct lysp_ext_instance {
     const char *name;                       /**< extension identifier, including possible prefix */
     const char *argument;                   /**< optional value of the extension's argument */
-    LY_VALUE_FORMAT format;                /**< prefix format of the extension name/argument (::LY_VALUE_XML is YIN format) */
+    LY_VALUE_FORMAT format;                 /**< prefix format of the extension name/argument (::LY_VALUE_XML is YIN format) */
+    struct lysp_node *parsed;               /**< Simply parsed (unresolved) YANG schema tree for ::lyplg_ext.compile callback,
```

Comment:
The comment should probably say that it serves as a cache and is filled only if `lys_compile_extension_instance()` is called. It should not be needed to mention that it must not be modified because that is true for all of these members and understood implicitly.